es6-classes/circle.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(radius) {
+    super();
+    this.radius = radius;
+    this.perimeter = 2 * Math.PI * radius;
+    this.area = Math.PI * radius ** 2;
```

Comment:
Instead of creating `perimeter` & `area` directly here, you should be calling `super` to generate these properties